app/mongock/changesets/CacheChangelog.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mongock.changesets;
+
+import com.github.cloudyrock.mongock.ChangeLog;
+import com.github.cloudyrock.mongock.ChangeSet;
+import com.mongodb.BasicDBObject;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+
+import java.time.Instant;
+import java.util.Date;
+
+@ChangeLog
+public class CacheChangelog {
+    private String collection = "cache";
+
+    @ChangeSet(order = "001", id = "Internal Movements DB Baseline", author = "Steve Sugden")
+    public void dbBaseline(MongoDatabase db) {
+        Document query = new Document();
```

Comment:
If this works, it will add a new field to existing cache documents (that doesn't exist in the model).
We can then safely delete lines 34-36.